src/sas/qtgui/MainWindow/DataExplorer.py

```diff
@@ -616,8 +616,22 @@ def updatePerspectiveWithProperties(self, key, value):
         if 'pr_params' in value:
             self.cbFitting.setCurrentIndex(self.cbFitting.findText('Inversion'))
             params = value['pr_params']
-            self.sendItemToPerspective(items[0])
-            self._perspective().updateFromParameters(params)
+            # Make the perspective read the rest of the read data
```

Comment:
this conditional contains the same content as the branch above (for fitting) and the common code should be merged